app/models/chargeback_rate.rb

```diff
@@ -70,4 +72,10 @@ def self.seed
       end
     end
   end
+
+  def assigned?
+    assigned_tos = get_assigned_tos
+    assigned_tos == {:objects=>[], :tags=>[]} ? false : true
```

Comment:
The `? false : true` is not necessary since the equality operator returns a boolean, so this can be simplified to:

```ruby
get_assigned_tos != {:objects=>[], :tags=>[]}
```